.changeset/new-rules-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-github': patch
+---
+
+add description onTeamEditedInOrganization event
```

Comment:
```suggestion
Added description to the `onTeamEditedInOrganization` event
```
Is this `onTeamEditedInOrganization` really the event name? Otherwise feel free to substitute this with the correct event name :)